docs/sequence/PISPDiscoveryApi.puml

```diff
@@ -0,0 +1,143 @@
+@startuml
+
+title PISP Discovery
+
+participant "PISP Backend" as PISP
+box "PISP tp-scheme-adapter"
+  participant "outbound-server" as PISP_TP_OUT
+  participant "PISPDiscoveryModel" as PISP_DM
+  participant "inbound-server" as PISP_TP_IN
+end box
+box "Mojaloop"
+    participant Switch
+end box
+box "DFSP tp-scheme-adapter"
+  participant "inbound-server" as DFSP_TP_IN
+  participant "DFSPDiscoveryModel" as DFSP_DM
+end box
+
+== Discovery ==
+autonumber 1 "<b>DISCOVERY-#</b>"
+rnote right of PISP
+PISP presents thirdparty enabled DFSPs to user. User selects a dfpsa and provides a generic ID to search for associated accounts
+end note
+PISP -> PISP_TP_OUT: GET /linking/accounts/dfspa/username1234
+
+activate PISP
+activate PISP_TP_OUT
+
+PISP_TP_OUT -> PISP_TP_OUT: const model = await create()
+rnote right of PISP_TP_OUT: state: start
+PISP_TP_OUT -> PISP_DM: model.getAccounts()
```

Comment:
let signal we are passing these two required arguments to model
```suggestion
PISP_TP_OUT -> PISP_DM: model.getAccounts(dfspId, username)
```

Reply:
done